src/GuzzleClientFactory.php

```diff
@@ -104,7 +97,8 @@ private function createDefaultDecider($maxRetries)
                 $this->logger->warning(
                     sprintf(
                         'Request failed (%s), retrying (%s of %s)',
-                        empty($error) ? $response->getBody()->getContents() : $error->getMessage(),
+                        empty($error) ? empty($response) ? 'No error' :
```

Comment:
Vnoreny ternar :scream: Jako klidne jo, ale uzavorkuj to a nejak lip odradkuj prosim :pray: